src/EnergyPlus/SolarShading.hh

```diff
@@ -542,6 +545,7 @@ struct SolarShadingData : BaseGlobalStruct
         this->ShadowingDaysLeft = 0;      // Days left in current shadowing period
         this->debugging = false;
         this->GetInputFlag = true;
+        this->GetInputFlagforSurfGeom = true;
         this->firstTime = true;
```

Comment:
Add a separate flag for the added simplified input processing function, so no interfering with the main one.

Reply:
New implementation: the `anyScheduledShadingSurface` flag